README.md

```diff
@@ -29,7 +29,14 @@ dependencies and passes them to a library called [Dependency-Check](https://gith
 
 <details>
 
-Please create a separate project consisting exclusively of `[nvd-clojure/nvd-clojure "2.2.0"]`. Said project can be located inside the targeted repo's Git repository.
+Please create a separate project consisting of `[nvd-clojure/nvd-clojure "2.2.0"]`. Said project can be located inside the targeted repo's Git repository.
+
+```
+(defproject nvd-helper "local"
+  :description "nvd-clojure helper project"
+  :dependencies [[nvd-clojure "2.2.0"]
+                 [org.clojure/clojure "1.10.3"]])
```

Comment:
On second thoughts, you can remove `[org.clojure/clojure "1.10.3"]`, while also changing in project.clj:

```diff
-[org.clojure/clojure "1.10.3" :scope "provided"]
+[org.clojure/clojure "1.10.3"]
```